### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by bpradipt**

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/12#" title="Author self-approved">bpradipt</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: scripts/images/convert_qcow2_ova.py
Comment (mkumatag) [inline scripts/images/convert_qcow2_ova.py:453]: will have to find a better way with other libraries to check the required args, will take that as a separate task,!

**Review by mkumatag — APPROVED** [review]
LGTM, thanks for the PR

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/12#" title="Author self-approved">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/12#pullrequestreview-492021250" title="Approved">mkumatag</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: scripts/images/convert_qcow2_ova.py
Comment (bpradipt) [reply scripts/images/convert_qcow2_ova.py:453]: Agree.. this is ugly.. We should switch to `argparse` going forward

**Pull request merged**
